services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/LegAnnotation.java

```diff
@@ -219,6 +223,28 @@ public abstract static class Builder extends DirectionsJsonObject.Builder<Builde
     @NonNull
     public abstract Builder trafficTendency(@Nullable List<Integer> trafficTendency);
 
+    /**
+     * Use this method to add parameters which are not present in the model yet but are supported
+     * on the Directions API side in the response.
+     * Use it for experimental parameters.
+     *
+     * @param unrecognizedProperties parameters to add to request
+     */
+    @NonNull
+    public Builder unrecognizedProperties(
```

Comment:
```suggestion
    internal Builder unrecognizedProperties(
```
I believe users are not supposed to use it, aren't they?

Reply:
it's a fair point